tests/models/test_validator.py

```diff
@@ -0,0 +1,145 @@
+"""Tests for the event validator"""
+
+import json
+import logging
+
+import pytest
+
+from ralph.exceptions import BadFormatException, UnknownEventException
+from ralph.models.selector import ModelSelector
+from ralph.models.validator import Validator
+
+from tests.fixtures.edx.browser import PageCloseBrowserEventFactory
+from tests.fixtures.edx.server import ServerEventFactory
+
+
+@pytest.mark.parametrize("ignore_errors", [True, False])
+@pytest.mark.parametrize("fail_on_unknown", [True, False])
+def test_models_validator_validate_with_no_events(
+    ignore_errors, fail_on_unknown, caplog
+):
+    """Tests given no events, the validate method does not write error messages."""
```

Comment:
```suggestion
    """Given no events the validate method does not write error messages."""
```

Reply:
When comparing with `test_selector` docstrings, I think it is better to get the same sentence structure. WDYT?